src/main/resources/forge.exc

```diff
@@ -66,4 +66,6 @@ net/minecraft/command/EntitySelector.matchEntitiesDefault(Lnet/minecraft/command
 net/minecraft/command/EntitySelector.matchesMultiplePlayersDefault(Ljava/lang/String;)Z=|p_82377_0_
 net/minecraft/command/EntitySelector.isSelectorDefault(Ljava/lang/String;)Z=|p_82378_0_
 
-net/minecraft/client/util/RecipeItemHelper.accountStack(Lnet/minecraft/item/ItemStack;I)V=|p_194112_1_,forceCount
\ No newline at end of file
+net/minecraft/client/util/RecipeItemHelper.accountStack(Lnet/minecraft/item/ItemStack;I)V=|p_194112_1_,forceCount
+net/minecraft/client/renderer/ActiveRenderInfo.updateRenderInfo(Lnet/minecraft/entity/player/EntityPlayer;Z)V=|p_74583_0_,p_74583_1_
+net/minecraft/client/renderer/ActiveRenderInfo.updateRenderInfo(Lnet/minecraft/entity/Entity;Z)V=|p_74583_0_,p_74583_1_
```

Comment:
I recommend putting a linebreak at the end, so the last line is not changed (like above) when a new line is added.